modules/core/src/utils/logtagconfigparser.hpp

```diff
@@ -0,0 +1,54 @@
+// This file is part of OpenCV project.
```

Comment:
Tabs in source files (logtagconfigparser.hpp, cpp) will be changed to whitespace in the next commit.
```
modules/core/src/utils/logtagconfigparser.cpp:18: tab in indent.
+	parse(input);
modules/core/src/utils/logtagconfigparser.cpp:27: tab in indent.
+	m_input = input;
modules/core/src/utils/logtagconfigparser.cpp:28: tab in indent.
+	segmentTokens();
modules/core/src/utils/logtagconfigparser.cpp:29: tab in indent.
+	return (m_malformed.empty());
modules/core/src/utils/logtagconfigparser.cpp:55: tab in indent.
...
```